ci-operator/config/3scale-qe/3scale-deploy/3scale-qe-3scale-deploy-main__3scale-amp-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,61 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  test-image:
+    name: 3scale-interop-tests
+    namespace: ci
+    tag: v2.13
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: 3scale-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 3scale-amp-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      AWS_REGION: us-east-2
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      DEPL_BUCKET_NAME: 3scale-apimanager-s3-bucket
+      DEPL_PROJECT_NAME: threescale
+      OPERATORS: |
+        [
+            {"name": "3scale-operator", "source": "redhat-operators", "channel": "!default", "install_namespace": "threescale", "target_namespaces": "threescale"}
+        ]
+    post:
```

Comment:
@ascerra and I had a discussion yesterday about how we should handle modifying the ipi-aws workflow. We thought it might be easiest to create your own workflow.  The main reason being, if something changes in the ipi-aws workflow, and we are testing multiple versions of a product, we would have to make changes in multiple config files, rather than just one workflow file. Any thoughts on that?

Here is how I did it in OADP: https://github.com/openshift/release/tree/master/ci-operator/step-registry/oadp/ipi/aws

Reply:
@calebevans I agree with creating our own workflow as the case you mentioned it would be easy to update only workflow. I'll update my PR with these changes